lets-encrypt/activate-ssl.sh

```diff
@@ -242,7 +245,7 @@ fi
 # Check if $certfiles exists
 if [ -d "$certfiles" ]; then
 # Activate new config
-        bash /var/scripts/test-new-config.sh
+        bash /var/scripts/test-new-config.sh $domain.conf
```

Comment:
`test-new-config.sh` is a totally different script that tests the configuration and has nothing to do with the domain name. @Pe46dro Please explain this fix.